pkg/cache/node_events.go

```diff
@@ -14,27 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package client
+package cache
```

Comment:
This change looks weird: I think it is github folding a delete and add into one change.

Reply:
the diff shows this correctly